src/src/jwt.ts

```diff
@@ -0,0 +1,32 @@
+import { Router } from 'express';
+
+const router = Router();
+export default router;
+
+const jws = require('jws');
+const fs = require('fs');
+const expect = require('chai').use(require('chai-subset')).expect;
```

Comment:
`chai` is a test assertion library - I suggest to replace it with simple checks.